### PR TITLE
added dashboard widget

### PR DESCRIPTION
**Pull request by slambert**
Starting to address #20. Buries it in bottom left column, but it's there. Will put effort into improving it if it ends up being useful, or abandon this effort and remove it.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -50,6 +50,34 @@ function c4aa_add_body_class_from_acf_options( $classes ) {
...
+
+add_action('wp_dashboard_setup', 'c4aa_custom_dashboard_widgets');
+  
+function c4aa_custom_dashboard_widgets() {
+global $wp_meta_boxes;
+ 
+wp_add_dashboard_widget('custom_help_widget', 'C4AA Theme Notes', 'custom_dashboard_help');
+}
+ 
+function custom_dashboard_help() {
```
Comment (laras126) [inline functions.php]: @slambert Add c4aa_ name space to this function also, so `c4aa_dashboard_help`, then this is good!

Comment (slambert) [reply functions.php]: Done - thanks!

**Comment by slambert** [comment]
This should be ready now.

File: functions.php [hunk 2]
```diff
@@ -50,6 +50,34 @@ function c4aa_add_body_class_from_acf_options( $classes ) {
...
+ * A place for specific reference info & instructions about the theme 
+ * for users of the site.
+ * 
+ */ 
+
+
+add_action('wp_dashboard_setup', 'c4aa_custom_dashboard_widgets');
+  
+function c4aa_custom_dashboard_widgets() {
+global $wp_meta_boxes;
```
Comment (laras126) [inline functions.php]: @slambert The contents of the functions should be indented one tab within the curly braces for readability - just an FYI, will approve and merge.

**Pull request merged**